examples/benchmark/deserialize_benchmark.py

```diff
@@ -0,0 +1,172 @@
+import torch
+import shutil
+import os
+import time
+import subprocess
+from pathlib import Path
+from tensorizer import TensorDeserializer
+from tensorizer.utils import convert_bytes, get_mem_usage
+import numpy as np
+
+# disable missing keys and unexpected key warnings
+os.environ["TRANSFORMERS_VERBOSITY"] = "error"
+
+# Improve safetensors performance
+os.environ["SAFETENSORS_FAST_GPU"] = "1"
+
+from accelerate import init_empty_weights
+from transformers import AutoModelForCausalLM, AutoTokenizer, AutoConfig
+
+
+MODEL_PATH = Path(os.environ.get("MODEL_PATH", f"./models"))
+TZR_PATH = MODEL_PATH / "model.tensors"
+HF_PATH = MODEL_PATH / "hf"
+RES_PATH = Path(os.environ.get("RES_PATH", "./results"))
+NUM_TRIALS = int(os.environ.get("NUM_TRIALS", 1))
+SKIP_HF = os.environ.get("SKIP_HF", False)
+SKIP_TZR = os.environ.get("SKIP_TZR", False)
+SKIP_ST = os.environ.get("SKIP_ST", False)
```

Comment:
Same casting deal here, this can also confuse people who would want to use these env vars and the only way to actually set them to `False` is to remove the definition of these env vars from the manifest launching this script.